src/framework/interface_message_box.sh

```
@@ -0,0 +1,29 @@
+
+function interface_message() {
+	# Check if argument is provided
+	if [[ -n "$1" ]]; then
+		input="$1"
+	else
+		input=$(cat)
+	fi
+	# Display menu based on DIALOG tool
+	case $DIALOG in
+		"dialog")
+		dialog --title "$TITLE" --msgbox "$input" 0 0 2>&1 >/dev/tty
```

Shellcheck (0.10.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)